api/src/main/java/io/opentelemetry/trace/SpanId.java

```diff
@@ -139,6 +150,14 @@ public String toLowerBase16() {
     return new String(chars);
   }
 
+  static SpanId generateRandomId(Random random) {
```

Comment:
nit: Consider to resolve this in a separate PR :)

Reply:
SG, reverted in https://github.com/open-telemetry/opentelemetry-java/commit/d912ead5b7bc04594c8300f690e6faff51fa760e. This will be done in a separate PR.